src/App.js

```diff
@@ -107,11 +107,22 @@ class App extends Component {
     if (this.hasWhitelistedScenes()) {
       this.requestScenesStatus();
     }
+    var intervalId = setInterval(this.timer, 5000);
```

Comment:
please use `const` (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) or just dont declare it and do eg
`this.setState({intervalId: setInterval(this.timer, 5000)});`

Reply:
done